templates/patterns/dropdown/pattern-dropdown.html.twig

```diff
@@ -14,4 +14,4 @@
   'remove_wrapper': remove_wrapper,
```

Comment:
We have a problem with trigger. He is waiting a button or link "component" but nobody has to know how is a component internally so we need the user here put a button or link pattern and map to adapt to component.